package.json

```
@@ -37,6 +42,9 @@
     "react-router-dom": "5.2.0"
   },
   "devDependencies": {
+    "@cypress/webpack-preprocessor": "^5.9.1",
+    "cypress": "^8.3.1",
+    "cypress-multi-reporters": "^1.5.0",
```

don't we need eslint plugin ?
```suggestion
    "@cypress/webpack-preprocessor": "^5.9.1",
    "cypress": "^8.3.1",
    "cypress-multi-reporters": "^1.5.0",
    "eslint-plugin-cypress": "^2.12.1",
```

---

Present here: https://github.com/SanjalKatiyar/dfr-console/blob/master/package.json#L66